armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -182,7 +200,28 @@ EOF
 
 
 # OS CONFIG --------------------------------------------------------------------
-cat << EOF > /root/.bashrc-custom
+# customize MOTD
+echo "MOTD_DISABLE='header tips updates armbian-config'" >> /etc/default/armbian-motd
+cat << EOF > /etc/update-motd.d/20-shift
+#!/bin/bash
+. /etc/os-release
```

Comment:
(Optional) I find it more readable to use `source` instead of `.`:

```
source /etc/os-release
source /etc/armbian-release
```

Reply:
That's the Armbian standard, but I completely agree. Changed to `source`.